cmake/resources/windows.rc.in

```diff
@@ -0,0 +1,34 @@
+#include <winver.h>
+
+IDI_ICON1   ICON    "@RES_DIR@/icon.ico"
+
+VS_VERSION_INFO VERSIONINFO
+    FILEVERSION        @PROJECT_VERSION_MAJOR@,@PROJECT_VERSION_MINOR@,@PROJECT_VERSION_PATCH@,@PROJECT_VERSION_TWEAK@
+    PRODUCTVERSION     @PROJECT_VERSION_MAJOR@,@PROJECT_VERSION_MINOR@,@PROJECT_VERSION_PATCH@,@PROJECT_VERSION_TWEAK@
+    FILEFLAGSMASK      VS_FFI_FILEFLAGSMASK
+    FILEFLAGS          (VS_FF_DEBUG|VS_FF_SPECIALBUILD)
```

Comment:
Is `VS_FF_DEBUG` always true? At least in release mode, no debug information is generated. In `RelWithDebInfo`, only a `pdb` is generated but the code itself if optimized and `NDEBUG` was defined in the build (i.e. it wasn't compiled with debugging features enabled nor does _this_ file contain debug information).

Reply:
i wrongly assumed RelWithDebInfo has at least some kind of symbols left unstriped. 
Removed the flag as there is no point check for debug builds specifically if we dont ship such.